app/src/main/java/com/github/houseorganizer/houseorganizer/SettingsActivity.java

```diff
@@ -1,17 +1,29 @@
 package com.github.houseorganizer.houseorganizer;
 
+import android.content.SharedPreferences;
 import android.os.Bundle;
 
 import androidx.appcompat.app.ActionBar;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.preference.Preference;
 import androidx.preference.PreferenceFragmentCompat;
+import androidx.preference.PreferenceManager;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.firestore.DocumentReference;
```

Comment:
there's 3 unused imports here, would be nice to clean them now

Reply:
Fixed